Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/CovidCertificateConfigMapper.kt

```diff
@@ -88,4 +101,8 @@ class CovidCertificateConfigMapper @Inject constructor() : CovidCertificateConfi
         override val waitAfterPublicKeyRegistration: Duration = Duration.standardSeconds(10),
         override val waitForRetry: Duration = Duration.standardSeconds(10),
     ) : CovidCertificateConfig.TestCertificate
+
+    companion object {
+        private val DEFAULT_EXPIRATION_THRESHOLD: Duration get() = Duration.standardDays(14)
```

Comment:
why `get()` it does not change, right?

Reply:
Yeah, but will most likely never be called.

Reply:
what do you mean by `never be called` ? why have it then ?

Reply:
It can be called under some circustances, like when the param expirationThresholdInDays is 0 or missing. I only adjusted how this gets called..